### PR TITLE
Js popup window

### PR DESCRIPTION
**Pull request by Enning94**
### JS popup window for personal portfolio-Figma template 2: Mobile and Desktop version

In this milestone, my partner and i:

-Added a popup window to the website
-Updated the page to render HTML information with javascript.

### CODE REVIEW

File: project-card.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+];
```
Comment (roshan-bajgain) [inline project-card.js:50]: - [ ] Your design is not similar to the Figma design kindly make it similar to the design.
- [ ] kindly fix the buttons as well.
### Your design
figma

File: project-card.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
```
Comment (roshan-bajgain) [inline project-card.js:49]: - [ ] Your popup is working properly but it effect the design as the design has more spacing on right kindly fix it
### Your design
his

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Enning94, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: project-card.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                  </div>
+                  
+                <div class='modal-paragraph-container'>
+                  <div class='modal-paragraph'>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry
+                    . Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a 
+                    galley of type and scrambled it 1960s with the releax map lapora verita.
+                  </div>
+                  <div class='modal-buttons-cont'>
+                    <button type='button' class='btn btn-danger model-button '><a href='${project[0].liveVersion}'>See Live </a><i class='fa fa-external-link' aria-hidden='true'></i></button>
```
Comment (leonardodiasb) [inline project-card.js:114]: - [ ] Kindly fix the sizing of your image when the screen size changes. In some screen sizes, your popup doesn't look like the Figma design and makes the popup taller than the screen height.
- [ ] Kindly fix the popup sizing in the mobile view. The popup is larger and taller than the mobile screen size and this is making a bad user experience due to the scroll bars. Please fix that.
Your design:
image
image
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Enning94, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @Enning94, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 
- [ ] Fllow Github flow :heavy_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**